vgstation13.dme

```diff
@@ -464,8 +464,8 @@
 #include "code\datums\wires\wires.dm"
 #include "code\defines\byondtools.dm"
 #include "code\defines\obj.dm"
-#include "code\defines\vox_sounds.dm"
 #include "code\defines\vox_sound_lengths.dm"
+#include "code\defines\vox_sounds.dm"
```

Comment:
🤔

Reply:
No changes were made to that file (you can see it's both "removed" and "added".

Just a compiler quirk.

Reply:
It always sorts them alphabetically
This happens when someone adds a file manually rather than by ticking the box in DM but doesn't place it in the right place